src/utils/index.js

```diff
@@ -26,6 +27,15 @@ function clearCanvas() {
   context.clearRect(0, 0, canvas.width, canvas.height)
 }
 
+function isSupportedEmoji(unified) {
+  let unifiedAsNum = parseInt(unified, 16);
+  let isSupported = ranges.filter(range => {
```

Comment:
I'd have written this with a for loop. Using a filter means it has to needlessly check the rest of the ranges even when it has a hit